<ui version="4.0" >
 <class>SpellCheckConfig</class>
 <widget class="QWidget" name="SpellCheckConfig" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>259</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Spell Check Configuration</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label2" >
       <property name="text" >
        <string>Default Language:</string>
       </property>
       <property name="buddy" >
        <cstring>language</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="language" />
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Options</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QCheckBox" name="bgSpellCheck" >
        <property name="text" >
         <string>Enable &amp;background spellchecking</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="skipUppercase" >
        <property name="text" >
         <string>Skip all &amp;uppercase words</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="skipRunTogether" >
        <property name="text" >
         <string>S&amp;kip run-together words</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="title" >
      <string>Ignore these words</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" colspan="2" >
       <widget class="QLineEdit" name="ignoreLineEdit" />
      </item>
      <item rowspan="3" row="1" column="0" >
       <widget class="QListWidget" name="ignoreList" />
      </item>
      <item row="1" column="1" >
       <widget class="QPushButton" name="add" >
        <property name="text" >
         <string>&amp;Add</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QPushButton" name="remove" >
        <property name="text" >
         <string>&amp;Remove</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
